.github/scripts/generate_binary_build_matrix.py

```diff
@@ -219,9 +219,10 @@ def generate_wheels_matrix(os: str,
                         "container_image": WHEEL_CONTAINER_IMAGES[arch_version],
                         "package_type": package_type,
                         "pytorch_extra_install_requirements":
-                        "nvidia-cuda-runtime-cu11;"
+                        "nvidia-cuda-runtime-cu11==11.7.99;"
                         "nvidia-cudnn-cu11==8.5.0.96;"
-                        "nvidia-cublas-cu11==11.10.3.66",
+                        "nvidia-cublas-cu11==11.10.3.66;"
+                        "nvidia-nccl-cu11==2.14.3",
                         "build_name":
```

Comment:
Should we also add libnvrtc here ? as in this PR https://github.com/pytorch/builder/pull/1183 
And remove libnvrtc from patch ?